vendor/github.com/prometheus/prometheus/promql/promqltest/testdata/duration_expression.test

```diff
@@ -145,4 +145,84 @@ eval instant at 1000s metric1_total offset -4
   metric1_total{} 100
 
 eval instant at 1000s metric1_total offset (-2 ^ 2)
-  metric1_total{} 100
\ No newline at end of file
+  metric1_total{} 100
```

Comment:
This change needs to be copied to `pkg/streamingpromql/testdata/upstream/duration_expression.test`

Reply:
I will leave it to @charleskorn or someone else from Query squad.

Reply:
Done in c39c5e6.